lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -302,6 +311,35 @@ END:
     type: process
     lambda: build-client-oauth-response
 
+CRI_TICF:
+  response:
+    type: process
+    lambda: call-ticf-cri
+    lambdaInput:
+      journeyType: main
+  parent: CRI_STATE
```

Comment:
Not sure of this parent ?

Reply:
Does the lambda call only respond `next` or `fail-with-ci`? 

If it's not getting events back from the CRI itself, it probably wont have the same event mappings as other cri states, which have defaults defined in the CRI_STATE parent:
```
CRI_STATE:
  events:
    next:
      targetState: UNEXPECTED_EVENT
    not-found:
      targetState: PYI_NO_MATCH
    ...
    vcs-not-correlated:
      targetState: PYI_NO_MATCH
```

I also think, in either case the name of the state might be misleading as it's of type process, rather than CRI, so maybe a better state name would be like "CALL_TICF_CRI" - a similar style to CHECK_EXISTING_IDENTITY which is also a process

Reply:
It can also respond with `/journey/error`, which we should handle. But I don't think it needs a parent.
And I agree with Mike about the naming thing.

Reply:
done